asset/scss/style.scss

```
@@ -0,0 +1,381 @@
+@import "variable";
+
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  font-family: "Montserrat", sans-serif;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+// mixin
+@mixin bgcolor {
+  background-color: $bg-color;
+  color: $white-color;
+}
+
+// main
+.main {
+  line-height: 1;
+  max-width: 1920px;
+}
+.title--left {
+  font-weight: 200;
+}
+.title--right {
```

nên đặt tên class có ý nghĩa hơn, ví dụ đặt theo format của bootstrap
https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics
```suggestion
.font-weight-200
.font-weight-800
```